### ISSUE TITLE
commit ab0a3b7 throw compilation error and warning

### ISSUE THREAD

**Issue by Canvis-Me** [Systems-Modeling/SysML-v2-Pilot-Implementation#423]
The latest changes to KerML.kerml and SysML.sysml threw an error and a warning respectively, is it a trivial error?

**Errors**
| Description | Resource | Path | Location | Type |
| --- | --- | --- | --- | --- |
| Couldn't resolve reference to Type 'double'. | KerML.kerml | /sysml.library/Kernel Libraries/Kernel Semantic Library | line: 388 /sysml.library/Kernel Libraries/Kernel Semantic Library/KerML.kerml | KerML Problem |

**Warnings**
| Description | Resource | Path | Location | Type |
| --- | --- | --- | --- | --- |
| Redefining feature should not have smaller multiplicity lower bound | SysML.sysml | /sysml.library/Systems Library | line: 338 /sysml.library/Systems Library/SysML.sysml | SysML Problem |

**Comment by seidewitz** [comment]
These are due to know production glitches that should have been corrected before the release was tagged. In the first case, the type should be `Real` instead of `double`. In the second case, the multiplicity should be `0..1` instead of `1..1`.

Thanks for letting us know about this. I have pushed corrections to the files to the master branch in the SysML v2 Pilot Implementation repository. The corrected versions will also be included in the public release (to be announced tomorrow).
